eng/testing/tests.singlefile.targets

```diff
@@ -0,0 +1,62 @@
+<Project>
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+
+    <BundleDir>$([MSBuild]::NormalizeDirectory('$(OutDir)', 'publish'))</BundleDir>
+    <RunScriptOutputPath>$([MSBuild]::NormalizePath('$(BundleDir)', '$(RunScriptOutputName)'))</RunScriptOutputPath>
+    <PublishSingleFile>true</PublishSingleFile>
+    <UseAppHost>true</UseAppHost>
+    <RuntimeIdentifier>$(PackageRID)</RuntimeIdentifier>
+    <SelfContained>true</SelfContained>
+    <SingleFileHostSourcePath>$(ArtifactsBinDir)$(RuntimeIdentifier).$(Configuration)/corehost/singlefilehost</SingleFileHostSourcePath>
```

Comment:
Is that path already set in the host build infra and could we re-use it to avoid the hard-coding here?